Gemfile

```diff
@@ -10,7 +10,7 @@ gem "decidim", git: "https://github.com/decidim/decidim.git", branch: DECIDIM_VE
 
 gem "decidim-decidim_awesome", "~> 0.7.0"
 gem "decidim-homepage_interactive_map", git: "https://github.com/OpenSourcePolitics/decidim-module-homepage_interactive_map.git", branch: "master"
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
+# gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
```

Comment:
Hi @eliegaboriau, is there a reason why you removed the term customizer gem?

Reply:
I removed it cause it wasn't the good branch, it should be okay now